### ISSUE TITLE
Default forms for creating child Places should not allow changing the parent place

### ISSUE THREAD

**Issue by estellecomment** [medic/medic-webapp#2725]
In new People tab, you create Persons and Places from their parent place. Thus the form should get the value of the parent passed in, display it, and not allow changing it. 

Currently, default forms allow changing the parent : 
Health_center : 
![screen shot 2016-09-09 at 11 49 35](https://cloud.githubusercontent.com/assets/911434/18382984/a9dda730-7683-11e6-845c-1c7182890cb1.png)

Clinic : 
![screen shot 2016-09-09 at 11 50 02](https://cloud.githubusercontent.com/assets/911434/18383011/ca80d412-7683-11e6-94c4-3a02c0b227fa.png)

**Comment by estellecomment** [comment]
Blocked on mocks for forms (to make sure we're actually displaying the parent place in the end!) : https://github.com/medic/medic-webapp/issues/2724

**Comment by sglangevin** [comment]
Closing in favor of #2828, which has complete requirements for default forms.
